apps/utils/scripts/run-db-migrator

```diff
@@ -238,7 +238,7 @@ fi
 
 # Set up the configuration variables using the setvar function.
 export BFD_ENV_NAME=laptop
-setvar db/url "jdbc:postgresql://${db_host}:${db_port}/bfd"
+setvar db/url "jdbc:postgresql://${db_host}:${db_port}/fhirdb"
```

Comment:
the wiki instructions need to be updated to use `fhirbd` for [Testing Local Environment Setup](https://github.com/CMSgov/beneficiary-fhir-data/wiki/Testing-Your-Local-Environment-Setup) once this is approved

Reply:
Looks good! I think we just need to make the same change for `run-bfd-pipeline` and then we should be set.